bqschema_suite_test.go

```diff
@@ -1,10 +1,10 @@
-package bqschema_test
+package bqschema
```

Comment:
I have the above commit ready, but I chose not to push it pending further discussion.